SU-HSDO/suhumsci
# SHS-4929: Hide caption/credits on images in default views

# READY FOR REVIEW

## Summary
Hide caption credits on images in default views

## Need Review By (Date)
26/07

## Urgency
medium

## Steps to Test
1. On Colorful and Traditional
2. Import configuration
3. Go to the following pages on each site
    - /default-views/news
    - /default-views/publications
    - /default-views/people
4. Check the caption/credits on images are hidden

## Steps to update configuration to views on other existing sites
1. Go to the view of the site (E.g: https://chemistry.suhumsci.loc/, https://religiousstudies.suhumsci.loc/ ) 
2. In the `Fields` section, select the `Image` kind of field
3. Change the configuration of the `View mode` from `Caption/Credit` to `Responsive Lazy`
4. Save it and check caption/credits on images are hidden

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207827270329900
